dev-python/importlib-resources/importlib-resources-6.4.0.ebuild

```diff
@@ -5,7 +5,7 @@ EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
 # backport from py3.12
```

Comment:
I'm not against adding it - because it is actually continuing past 3.12, given people are going to keep using it whether we like it or not (although we should try fix stuff upstream when they do), but the comment becomes either misleading or useless now.